packages/financial-contracts/schema.graphql

```diff
@@ -33,15 +35,9 @@ type FinancialContract @entity {
 
   cumulativeFeeMultiplier: BigDecimal
 
-  globalCollateralizationRatio: BigDecimal
-
-  totalSyntheticTokensCreated: BigDecimal!
```

Comment:
Tracking these is tough with the addition of the CFRM because `LiquidationCreated` for example emits the total amount of tokens liquidated as the funding-rate adjusted amount